crates/rpc/rpc-eth-api/src/core.rs

```diff
@@ -192,6 +192,15 @@ pub trait EthApi {
     #[method(name = "getHeaderByHash")]
     async fn header_by_hash(&self, hash: B256) -> RpcResult<Option<Header>>;
 
+    /// `eth_simulateV1` executes an arbitrary number of transactions on top of the requested state.
+    /// The transactions are packed into individual blocks. Overrides can be provided.
```

Comment:
```suggestion
    /// `eth_simulateV1` executes an arbitrary number of transactions on top of the requested state.
    /// The transactions are packed into individual blocks.
```
we don't need to document the rust API, at it's height we document how we use the rust API. or what else do you mean by "Overrides can be provided" if not default impl can be overridden?

Reply:
this refers to block and state overrides that are available on other endpoints